README.md

```diff
@@ -80,6 +80,34 @@ Integration with the Genius Yield DEX has never been easier.
 > [!TIP]
 > Have a look at sample configuration in [Building locally from source using the Haskell Toolchain](#building-locally-from-source-using-the-haskell-toolchain) section for thorough explanation of options made available to configure the server.
 
+### Setting up the API Server with Docker (Kupo)
```

Comment:
This is not present in ToC, also should this be nested further (i.e. `####`) as previous heading, "Spinning up the api server using docker" is also on setting up server with Docker.

Reply:
Addressed: e67838f0376f7fd40ae849f1591a0fb59c4325ca